doc/UsersGuide/BuildingRunningTesting/SRW-SD.rst

```diff
@@ -0,0 +1,277 @@
+   201907220000    upp_post_mem000_f000    18988282    SUCCEEDED            0       1      212.0
+   201907220000    upp_post_mem000_f001    18988283    SUCCEEDED            0       1      247.0
+   201907220000    upp_post_mem000_f002    18988284    SUCCEEDED            0       1      258.0
+   201907220000    upp_post_mem000_f003    18988285    SUCCEEDED            0       1      271.0
+   201907220000    upp_post_mem000_f004    18988286    SUCCEEDED            0       1      284.0
+   201907220000    upp_post_mem000_f005    18988287    SUCCEEDED            0       1      286.0
+   201907220000    upp_post_mem000_f006    18988288    SUCCEEDED            0       1      292.0
+   ==============================================================================================
+   201907220600              smoke_dust    18988289    SUCCEEDED            0       1      225.0
+   201907220600               prepstart    18988302    SUCCEEDED            0       1      112.0
+   201907220600           get_extrn_ics    18984150    SUCCEEDED            0       1       10.0
+   201907220600          get_extrn_lbcs    18984151    SUCCEEDED            0       1       14.0
+   201907220600         make_ics_mem000    18984188    SUCCEEDED            0       1      152.0
+   201907220600        make_lbcs_mem000    18984189    SUCCEEDED            0       1       79.0
+   201907220600         forecast_mem000    18988311    SUCCEEDED            0       1     6191.0
+   201907220600    upp_post_mem000_f000    18989105    SUCCEEDED            0       1      212.0
+   201907220600    upp_post_mem000_f001    18989106    SUCCEEDED            0       1      283.0
+   201907220600    upp_post_mem000_f002    18989107    SUCCEEDED            0       1      287.0
+   201907220600    upp_post_mem000_f003    18989108    SUCCEEDED            0       1      284.0
+   201907220600    upp_post_mem000_f004    18989109    SUCCEEDED            0       1      289.0
+   201907220600    upp_post_mem000_f005    18989110    SUCCEEDED            0       1      294.0
+   201907220600    upp_post_mem000_f006    18989111    SUCCEEDED            0       1      294.0
+
+If something goes wrong, users can check the log files, which are located by default in ``nco_dirs/test_smoke_dust/com/output/logs/20190722``.
+
+WE2E Test for SRW-SD
+=======================
+
+There is an SRW-SD Workflow End-to-End (WE2E) test in the SRW App repository that tests the configuration described in this chapter. To run the experiment as a WE2E tests, first build the app for SRW-SD:
+
+.. code-block:: console
+
+  ./app_build.sh -p=<machine>
+
+Add the WE2E test for SRW-SD to the list file:
+
+.. code-block:: console
+
+   cd /path/to/ufs-srweather-app/tests/WE2E
+   echo "smoke_dust_grid_RRFS_CONUS_3km_suite_HRRR_gf" >> my_tests.txt
+
+Run the WE2E test:
+
+.. code-block:: console
+
+   $ ./run_WE2E_tests.py -t my_tests.txt -m hera -a gsd-fv3 -q
```

Comment:
@MichaelLueken @BenKoziol-NOAA Should I remove this section of the docs? I don't think it works in `main_aqm`, and I'm not sure if the test in `develop` is comparable. When I run in the `main_aqm` branch, I get this: 
```
[Gillian.Petro@hfe05 WE2E]$ ./run_WE2E_tests.py -t my_tests.txt -m hera -a epic -q
Traceback (most recent call last):
  File "/scratch2/NAGAPE/epic/Gillian.Petro/ufs-srweather-app-sd/tests/WE2E/./run_WE2E_tests.py", line 13, in <module>
    from generate_FV3LAM_wflow import generate_FV3LAM_wflow
ModuleNotFoundError: No module named 'generate_FV3LAM_wflow'
```

Reply:
@gspetro-NOAA Yes, you should remove this section.  As you noted, only the `parm/config.aqm.yaml` and `parm/config.smoke_dust.yaml` sample configuration files will run in `main_aqm`.